solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LTRScoringModel.java

```diff
@@ -24,8 +24,8 @@
 import org.apache.lucene.index.LeafReaderContext;
 import org.apache.lucene.search.Explanation;
 import org.apache.solr.ltr.feature.FeatureException;
-import org.apache.solr.ltr.feature.norm.Normalizer;
-import org.apache.solr.ltr.feature.norm.impl.IdentityNormalizer;
+import org.apache.solr.ltr.norm.Normalizer;
+import org.apache.solr.ltr.norm.impl.IdentityNormalizer;
```

Comment:
I had wondered the same. Okay, will fold `norm.impl` into just `norm`. How about also folding `feature.impl` into `feature` (not as part of this pull request)?